src/client/debugger/jupyter/kernelDebugAdapter.ts

```diff
@@ -60,8 +60,6 @@ export class KernelDebugAdapter implements DebugAdapter, IKernelDebugAdapter, ID
         private fs: IFileSystem,
         private readonly kernel: IKernel | undefined
     ) {
-        void this.dumpAllCells();
```

Comment:
So in full debugging will I not be able to step into other cells?

Reply:
I tired it and it didn't make a difference, it always failed. Lets talk on Monday

Reply:
This PR fixes that:
https://github.com/microsoft/vscode-jupyter/pull/7591